### DISCUSSION TITLE
Having problem with the bot today

### DISCUSSION THREAD

**Discussion by FernandoLODamasceno** [jagrosh/MusicBot#1611] [General]
![image](https://github.com/jagrosh/MusicBot/assets/65792481/ad69b686-cd38-4830-bfbd-f945deb5c0bb)

This error popped up when i tried putting music.

**Comment by GioSaez** [comment]
Got another music bot that is also refusing to play music from youtube

**Comment by dogghoodie** [comment]
Same.

**Comment by MichailiK** [comment]
[JMusicBot 0.4.2](https://github.com/jagrosh/MusicBot/releases/tag/0.4.2) has been released with a fix for the "Not a success status code: 403" error.
